contracts/zaps/CheapestBondHelper.sol

```diff
@@ -1,56 +1,71 @@
 // SPDX-License-Identifier: GPL-2.0
-pragma solidity 0.7.5;
+pragma solidity ^0.8.0;
 
-interface IDepository {
-    function bondPriceInUSD() external view returns (uint price_);
-    function principle() external view returns (address);
-}
+import "./interfaces/IBondDepoV2.sol";
 
 contract CheapestBondHelper {
+    ////////////////////////// STORAGE //////////////////////////
 
-    //////////////// State ////////////////
+    /// @notice used for access control
+    address public olympusDAO = msg.sender;
```

Comment:
whats the diff between this and setting it in the constructor?

Reply:
there isn't one, but for the sake of tidiness I moved it to the constructor 🙂